.github/workflows/wordpress-plugin-deploy.yml

```diff
@@ -1,18 +1,41 @@
 name: Deploy to WordPress.org
```

Comment:
Not exactly sure how this plugin has been deployed to WordPress.org up until this point (doesn't seem like this Action was being used and I see there's a few bin scripts around building a release asset). I think the changes I've made here will make for a successful deploy but could use another eye on that

Reply:
@dsawardekar I believe you've done manual deploys for this, so hoping you can help comment/review these changes?

Reply:
@dkotter I was pushing to the .org SVN repo to deploy. The automatted deployment looks much better than the manual one. Let me know once released, and I can test on a few sites that I'm currently using it on.